task/backend/scheduler_test.go

```diff
@@ -456,13 +440,13 @@ func TestScheduler_Queue(t *testing.T) {
 	// Tick the scheduler so the next natural run will happen once 180 finishes.
 	o.Tick(3062)
 
-	// Cancel 180. Next run should be 3060, the next natural schedule.
+	// Cancel 180. Next run should be 240, manual runs get priority.
 	e.RunningFor(task.ID)[0].Cancel()
-	pollForRun(3060)
+	pollForRun(240)
 
 	// Cancel the 3060 run; 240 should pick up.
```

Comment:
Good catch on manual runs getting priority.  This and the next comment are now stale.